src/include/migraphx/context.hpp

```diff
@@ -172,6 +188,18 @@ struct context
         (*this).private_detail_te_get_handle().from_value(v);
     }
 
+    void set_exhaustive_tune_flag(bool v)
```

Comment:
This shouldn't be added to the `context` interface. The flag should just be added to the GPU context, and then we can set it in `get_passes`.

Reply:
Done.